### ISSUE TITLE
The oddities of the italic typeface

### ISSUE THREAD

**Issue by Antiever** [JetBrains/JetBrainsMono#570]
Here's a comparison of DejaVu Sans Mono vs JBM

![sublime_text_2022-10-22_08-03-09](https://user-images.githubusercontent.com/9666515/197321243-9cf8f415-cd49-4046-824a-1b74c266347f.png)

I think the first option is preferable

**Comment by philippnurullin** [comment]
Hi @Antiever . Thnaks for the feedback. 
Please send the actual symbols you using as text.

**Comment by alexeyten** [comment]
I would also check the version of the font and that there actually is italic font.

**Comment by YtotheWT** [comment]
However, in True italic, the `|` symbol is still tilted
![image](https://user-images.githubusercontent.com/62426149/197497059-6fa3ced9-a500-4dc0-b2ff-ae4458124402.png)

**Comment by philippnurullin** [comment]
Yes, `|` shoulden't be italicised. The update will come in the next push.

**Comment by Antiever** [comment]
> Please send the actual symbols you using as text

` ┌ ┬ ┐│├ ┤┼└ ┴ ┘`

They all shoulden't be italicised.
Thank you!

**Comment by philippnurullin** [comment]
None of these `┌ ┬ ┐│├ ┤┼└ ┴ ┘` are italicised in the font.

**Comment by Antiever** [comment]
In the screenshot the `bat` file and the table is the comment displayed by default in italic in text editors
